datastream-utils/src/main/java/com/linkedin/datastream/common/LogUtils.java

```diff
@@ -92,4 +95,44 @@ public static String logSummarizedTopicPartitionsMapping(List<String> partitions
             .append(logNumberArrayInRange(topicPartitionsMap.get(topicName))))
         .collect(Collectors.joining(", "));
   }
+
+  /**
```

Comment:
also lets have java docs explaining what each parameter is for eg. `contextPrefix`.